gpt4all-kotlin/src/jvmMain/kotlin/com/xebia/functional/gpt4all/port/KotlinPort.kt

```diff
@@ -0,0 +1,34 @@
+@file:JvmName("KotlinPort")
+
+package com.xebia.functional.gpt4all.port
+
+import com.xebia.functional.gpt4all.GPT4All
+import com.xebia.functional.xef.auto.AiDsl
+import com.xebia.functional.xef.auto.PromptConfiguration
+import com.xebia.functional.xef.prompt.Prompt
+import com.xebia.functional.xef.vectorstores.VectorStore
+import kotlinx.coroutines.reactive.asPublisher
+import org.reactivestreams.Publisher
+
+@AiDsl
+suspend fun promptStreaming(
```

Comment:
Should be in `xef-java` as operator similar how `promptMessages` and `prompt` are declared now.

Reply:
I tried to implement in CoreAIScope like promptMessages but I have to add the GPT4ALL dependency in xef-core

```kotlin
@AiDsl
suspend fun promptStreaming(
    gpt4all: GPT4All,
    question: String,
    context: VectorStore,
    promptConfiguration: PromptConfiguration = PromptConfiguration.DEFAULTS
): Publisher<String> = gpt4all.promptStreaming(Prompt(question), context, null, emptyList(), promptConfiguration).asPublisher()
```

Reply:
also I move KotlinPort.kt to xef-java but I'm having this error: 

```java
/java/auto/AIScope.java:15: error: package com.xebia.functional.xef.java.auto.port does not exist
import com.xebia.functional.xef.java.auto.port.KotlinPort;
```

Reply:
Implemented how promptMessages, Done